sme_finance_application_schema/translations.py

```diff
@@ -27,35 +27,81 @@ def finance_application_v3_to_sme_contact_v3(finance_application):
+                  'business_plan', 'card_revenue', 'online_revenue', 'institutional_revenue',
+                  'stock_ready', 'revenue_growth', 'intellectual_property', 'trade_credit',
+                  'business_premises', 'registered_brand', 'customers', 'region',
+                  'company_credit_rating', 'familiarity_with_financing', 'accounting_software'):
         if field in finance_application['requesting_entity']:
             sme_v5[field] = finance_application['requesting_entity'][field]
-    for field in ( 'requested_amount', 'finance_type_requested', 'date_finance_required',
-            'date_finance_requested', 'finance_term_length', 'guarantor_available', 'purpose'):
+    for field in ('requested_amount', 'finance_type_requested', 'date_finance_required',
+                  'date_finance_requested', 'finance_term_length', 'guarantor_available', 'purpose'):
         if field in finance_application['finance_need']:
             sme_v5[field] = finance_application['finance_need'][field]
     if finance_application.get('actors'):
         directors = [x for x in finance_application['actors'] if x['role'] == 'director']
-        if directors:
-            if len(directors) > 1:
-                raise Exception
-            director = directors[0]
+        for director in directors:
             if 'value_of_property_equity' in director:
-                sme_v5['directors_houses'] = director['value_of_property_equity']
+                director_equity = ['value_of_property_equity']
+                if sme_v5.get('directors_houses'):
+                    sme_v5['directors_houses'] += director_equity
+                else:
+                    sme_v5['directors_houses'] = director_equity
+
             if 'value_of_pension' in director:
-                sme_v5['directors_pensions'] = director['value_of_pension']
+                directors_pension = director['value_of_pension']
+                if sme_v5.get('directors_pensions'):
+                    sme_v5['directors_pensions'] += directors_pension
+                else:
+                    sme_v5['directors_pensions'] = directors_pension
+
             if 'familiarity_with_financing' in director:
-                sme_v5['familiarity_with_financing'] = director['familiarity_with_financing']
+                director_familiarity = director['familiarity_with_financing']
+
+                familiarity_table = {
+                    'first_time': 0,
+                    'had_finance_before': 1,
+                    'expert': 2,
+                }
+                reverse_familiarity_table = dict((reversed(item) for item in familiarity_table.items()))
+
+                if sme_v5.get('familiarity_with_financing'):
+                    max_familiarity = max(
+                        familiarity_table[director_familiarity],
+                        familiarity_table[director['familiarity_with_financing']]
```

Comment:
this doesn't make sense to me, aren't they the same value?

at the top of the if you are setting `director_familiarity = director['familiarity_with_financing']` then using that to access the table and at the same time doing `familiarity_table[director['familiarity_with_financing']]` this is going to result in the same value being passed into max twice, which makes it redundant.

If i'm reading the purpose correctly here wouldn't you be better off doing something like (not exactly):
```python
max_familiarity = max(
  familiarity_table[directory_familiarity],
  sme_v5['familiarity_with_financing']
)
```
I think you got this right in the credit rating section below